src/pages/projectSettings/tabs/NotificationsTab/NotificationsRow.tsx

```diff
@@ -0,0 +1,99 @@
+import styled from "@emotion/styled";
+import { NotificationMethod } from "components/Notifications/NotificationMethod";
+import { NotificationSelect } from "components/Notifications/NotificationSelect";
+import {
+  triggers,
+  subscriptionMethodControls,
+  subscriptionMethodDropdownOptions,
+} from "constants/triggers";
+import { useNotificationModal } from "hooks/useNotificationModal";
+
+import { Trigger } from "types/triggers";
+
+interface SubscriptionSelectProps {
+  resourceType;
+  trigger: Trigger;
+  // todo: use in future work
+  selectors;
+  regexSelectors;
+}
+
+export const SubscriptionSelect: React.VFC<SubscriptionSelectProps> = ({
+  trigger,
+}) => {
+  const type = "version";
+  const resourceId = "project-id";
+
+  const {
+    disableAddCriteria,
+    extraFieldInputVals,
+    extraFields,
+    onClickAddRegexSelector,
+    regexSelectorProps,
+    setExtraFieldInputVals,
+    setSelectedTriggerIndex,
+    showAddCriteria,
+    buildInitiatorSelected,
+    setBuildInitiatorSelected,
+  } = useNotificationModal({
+    subscriptionMethodControls,
+    triggers,
+    resourceId,
+    type,
+  });
+  const currentIndex = trigger.index || 0;
```

Comment:
Instead of setting `currentIndex` in this way, can you use the `selectedTriggerIndex` field returned by the `useNotifiationModal` hook [here](https://github.com/evergreen-ci/spruce/blob/2ae2fe4c34515896cd9d3b62a9e41a4cf071ed7e/src/hooks/useNotificationModal.ts#L261)?

Reply:
done